ipmof/geometry.py

```diff
@@ -16,12 +16,12 @@ class Coor(object):
     Coor class for holding 3D space coordinates.
     """
     def __init__(self, input):
-        if isinstance(input, list):
+        try:
             self.x = input[0]
             self.y = input[1]
             self.z = input[2]
-        else:
-            raise TypeError('Input type not supported. Use list [x, y, z]')
+        except:
+            raise TypeError('Use list, tuple, numpy array for coordinates [x, y, z]')
```

Comment:
either delete and let the original exception propagate, or use a more specific except, such as 'except TypeError

but probably you just want to let the original exception propagate.